vembrane/__init__.py

```diff
@@ -112,86 +151,134 @@ def get_annotation_keys(header: VariantHeader, ann_key: str) -> List[str]:
 
 @lru_cache(maxsize=32)
 def parse_annotation_entry(entry: str,) -> List[str]:
```

Comment:
since all this does is split, it should not be named parse :D

Reply:
Be my guest :).
Also, I don't think the LRU cache is really necessary.

Reply:
At least not anymore, since this doesn't do any parsing at all. And who knows if there are any identical annotation entries to start with (we should have checked the VCFs we have before putting this in)